content/en/monitors/guide/monitoring-available-disk-space.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Monitoring Available Disk Space
+kind: documentation
+---
+
+
+A common system metric to monitor is the available disk space on a given system or host. This guide will help you create a monitor alerting you when free disk space for a host falls below 10% for any host reporting to Datadog.
+
+To create the monitor for available disk space:
+
+1. In the navigation menu, click **Monitors**.
```

Comment:
```suggestion
1. Navigate to [Monitors][1].
```

And then add the following to the end of the doc:

[1]: https://app.datadoghq.com/monitors/